### PR TITLE
Fix TS build

### PR DESCRIPTION
**Pull request by JonTheBurley**
This PR fixes the rollup build to create proper typings for components. The previous plugin was not compatible with absolute imports. SVGs need to be imported relative though or they get treated as external files, so I changed some paths.

### CODE REVIEW

File: src/Filter/components/FilterSectionHeader/FilterSectionHeader.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { FC, MouseEventHandler } from 'react';
 import { FilterModule } from 'src/Filter/types';
-import { ReactComponent as Add } from 'src/Filter/assets/add.svg';
-import { ReactComponent as Remove } from 'src/Filter/assets/remove.svg';
+import { ReactComponent as Add } from '../../assets/add.svg';
+import { ReactComponent as Remove } from '../../assets/remove.svg';
```
Comment (jason-ransom-slalom) [inline src/Filter/components/FilterSectionHeader/FilterSectionHeader.tsx:4]: Can you add this to the contributing section of the README somewhere that makes sense? Would be good for future devs to know.

**Review by jason-ransom-slalom — APPROVED** [review]
Approved with recommendation on docs update.

File: src/Filter/components/FilterSectionHeader/FilterSectionHeader.tsx (hunk 1 above)
Comment (JonTheBurley) [reply src/Filter/components/FilterSectionHeader/FilterSectionHeader.tsx:4]: Sure, I'll do that now. Good idea.

**Review by michaelpuzon — APPROVED** [review]

**Review by adorack-tri — APPROVED** [review]
I'm OK with this, though the requirement to import SVGs as relative paths seems a bit arbitrary.  I don't have the background, though, on how TS is configured in the project.  Importing SVGs as absolute paths isn't just a matter of adding a rule in the webpack config?

**Comment by JonTheBurley** [comment]
@adorack-tri  There's no webpack in this project, just rollup. I will make an issue to investigate a bit further. When the rollup plugins go through everything, any absolute paths got treated as an external dependency instead of just a path to a place in the project but for SVGs only. There might be some configuration to be done in rollup's svgr or url plugins to fix this though. I spent a bit of time on it today, but it was proving to be more work than I had thought.

**Pull request merged**